src/Compatibility/ControlGallery/src/Android/Compatibility.ControlGallery.Android.csproj

```diff
@@ -62,4 +62,6 @@
     <Compile Include="Properties\MapsKey.cs" Condition="Exists('Properties\MapsKey.cs')" />
   </ItemGroup>
 
+  <Import Project="$(MauiSrcDirectory)Maui.InTree.props" Condition=" '$(UseMaui)' != 'true' " />
```

Comment:
All projects that need maui things, like xaml compilation, need to add `Maui.InTree.props` to the bottom of the .csproj in order to set values that are used by the nugets being imported.